.gitignore

```diff
@@ -1,6 +1,8 @@
-phpunit.result.cache
-.idea
+build/
 report/
 vendor/
+composer.lock
```

Comment:
I removed the composer.lock file from this repo. I'm not sure why it was committed? A composer.lock file is generated with environment info at the time dependencies were installed (like the PHP version). It's unsuited to be committed to be run as part of a CI build. I don't think it should stay hardcoded in a package repo.